README.md

```diff
@@ -73,3 +73,11 @@ Participation in the Argo CD project is governed by the [CNCF Code of Conduct](h
 1. [Getting Started with ArgoCD for GitOps Deployments](https://youtu.be/AvLuplh1skA)
 1. [Using Argo CD & Datree for Stable Kubernetes CI/CD Deployments](https://youtu.be/17894DTru2Y)
 
+
+## Vendor-hosted community resources
```

Comment:
This is a brief and non-comprehensive discussion among maintainers. Here is a brief high-level directive agreement.

- We don't want to rush to add the vendor section to the project's main page
- The decision to allow this will have a broader discussion among the community maintainers to reach an agreement
- If the maintainers agree to allow this, maintainers will also agree on strict guidance about those hyperlinks. Here is a preliminary proposal
> No login is required (No marketing-able information could be collected)
> No branding information (Are we ok for a vendor-specific domain to be used? Maybe Yes)
> Have to be up-to-date open source version with the latest patch (N, N-1) (If the instance is falling behind, it will be removed. If the instance is out of the security patch, it will be removed)
- This will be discussed in the coming maintainer meetings